naptime/src/main/scala/org/coursera/naptime/actions/RestActionBuilder.scala

```diff
@@ -167,7 +167,8 @@ class RestActionBuilder[RACType, AuthType, BodyType, ResourceKeyType, ResourceTy
               Status.BAD_REQUEST,
               Some("request.validation"),
               Some(e.getMessage),
-              None)
+              None,
```

Comment:
Let's follow the boy(or girl) scout rule of leaving the code better than you found it and use named parameters, especially for the `None` which can be anything. Also as we add more parameters it becomes a better idea to name at least some of them.